doc/classes/SpriteFrames.xml

```diff
@@ -39,6 +39,14 @@
 				Removes all animations. An empty [code]default[/code] animation will be created.
 			</description>
 		</method>
+		<method name="duplicate_animation">
+			<return type="void" />
+			<param index="0" name="anim_from" type="StringName" />
+			<param index="1" name="anim_to" type="StringName" />
+			<description>
+				Duplicates the animation [param anim_from] onto a new animation named [param anim_to]. Fails if [param anim_to] already exists.
```

Comment:
```suggestion
				Duplicates the animation [param anim_from] to a new animation named [param anim_to]. Fails if [param anim_to] already exists, or if [param anim_from] does not exist.
```
Onto denotes location, also should be obvious but if we mention the output animation people might get confused if we don't mention the other, so let's be explicit

Reply:
Well, I don't have much experience on how to use English idiomatically. I will just take your advice.